package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@kevquirk/simple.css",
+  "version": "1.0.0",
```

Comment:
note: as part of publishing, you'll need to bump the `version` each time.  There is a [CLI for that](https://docs.npmjs.com/cli/v6/commands/npm-version).

Reply:
What about 'simplecss' for the package name?

Reply:
Looks like it should be [available](https://www.npmjs.com/search?q=simplecss).  👍